<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GreyQuestion</class>
 <widget class="QDialog" name="GreyQuestion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>451</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>30</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="format">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="QuestionLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>371</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>Tady místo toho bude otázka</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Otázka:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAno">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Ano</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNe">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>220</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Ne</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
